src/com/company/Main.java

```diff
@@ -0,0 +1,73 @@
+package com.company;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        //homeTask 1
+
+        int i = 1;
+        while (i <= 10) {
+            System.out.print(i + " ");
+            i ++ ;
+        } {System.out.println(); }
```

Comment:
```suggestion
        }
    System.out.println();
```
скобки здесь немного избыточны